renovate.json

```
@@ -0,0 +1,6 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "config:base"
+  ]
```

```suggestion
  "extends": [
    "config:base",
    ":dependencyDashboard"
  ]
```

I suggest using the dashboard to have a central issue where to look at all proposed changes by Renovate.
We can also consider the manual aproval which would require approval to even open PR (but it might makes us update less often): https://docs.renovatebot.com/key-concepts/dashboard/

---

ignore this, it seems the dashboard is on by default

---

I think adding some sort of scheduling for the PRs could be usefull, lets say we want to merge deps PRs on fridays once a week [Reference](https://docs.renovatebot.com/configuration-options/#schedule)